lib/papyrus/state_machine.rb

```diff
@@ -32,7 +32,7 @@ module StateMachine
 
         def ___record__papyrus_state_machine_event(event, object)
           Thread.current['___papyrus_state_machine_events'] ||= []
-          Thread.current['___papyrus_state_machine_events'] << { event: event, object: object }
+          Thread.current['___papyrus_state_machine_events'] << {event: event, object: object, params: Papyrus.config.default_params(event, object)}
```

Comment:
Same, let's call it in Papyrus.event.
I've kept this block the same for nuntius/papyrus & vorto for consistency.

Reply:
did not add it to there so it could reflect "the owner/user of a transaction" as it was before. (transactionlog.user)
this is +better